PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcher.cpp

```diff
@@ -82,6 +82,39 @@ INT_PTR CALLBACK VerticalFileSwitcher::run_dlgProc(UINT message, WPARAM wParam,
 		{
 			switch (((LPNMHDR)lParam)->code)
 			{
+				case NM_CUSTOMDRAW:
+				{
+					static bool becomeDarkMode = false;
+					static bool becomeLightMode = false;
+					HWND hHeader = ListView_GetHeader(_fileListView.getHSelf());
+					if (NppDarkMode::isEnabled())
+					{
+						if (!becomeDarkMode)
+						{
+							NppDarkMode::setExplorerTheme(hHeader, false);
+							becomeDarkMode = true;
+						}
+						becomeLightMode = false;
+
+						auto nmtbcd = reinterpret_cast<LPNMTBCUSTOMDRAW>((LPNMHDR)lParam);
+						SetBkMode(nmtbcd->nmcd.hdc, TRANSPARENT);
+						FillRect(nmtbcd->nmcd.hdc, &nmtbcd->nmcd.rc, NppDarkMode::getBackgroundBrush());
+						nmtbcd->clrText = RGB(255, 255, 255);
+						SetTextColor(nmtbcd->nmcd.hdc, RGB(255, 255, 255));

```

Comment:
Won't it better to use `NppDarkMode::getTextColor();`

Reply:
These 2 lines have no effect ether. I will remove them.